MainBrain/src/Controller/CanController/CanController.cpp

```diff
@@ -95,6 +95,33 @@ void CanController::sendUnitekRead(uint8_t regId, uint8_t pollTime = 0)
     Serial.print(unitekMessage.buf[2]);
 }
 
+/** 
+ * @brief  formats and sends unitek can message to change the value of a register
+ * @note   
+ * @param  regID: register you want to change the value of
+ * @param  buf1: lower 8 bits of value
+ * @param  buf2: upper 8 bits of value
+ * @retval None
+ */
+void sendUnitekWrite(uint8_t regID, uint8_t buf1, uint8_t buf2){
+    //intializing and constructing the CAN message 
+    CAN_message_t canMessage;
+
+    canMessage.id=canModel.UNITEKSENDID;
+    canMessage.len=3;
+    canMessage.buf[0]=regID;
+    canMessage.buf[1]=buf1;
+    canMessage.buf[2]=buf2;
```

Comment:
Should these be in reverse byte order, based on last semester testing?

Reply:
I figured they would be put into reverse byte order as parameters to the function